src/utils/searchViewHelper.js

```diff
@@ -31,9 +32,11 @@ const searchViewHelper = {
     // If repository is selected after search value is input, fetch search results
     if (search !== '') {
       searchViewHelper.searchByRevisionOrEmail(repository, search);
+    } else {
+      // Fetch 10 most recent revisions when repository changes
+      store.dispatch(fetchRecentRevisions(repository));
     }
-    // Fetch 10 most recent revisions when repository changes
-    store.dispatch(fetchRecentRevisions(repository));
+    store.dispatch(updateSearchIsFocused(true));
```

Comment:
set input to be focused, so that recent revisions will display when a repository is selected